src/caselawclient/models/documents/__init__.py

```diff
@@ -566,3 +566,13 @@ def linked_documents(self, namespaces: list[str], only_published: bool = True) -
     def content_as_html(self) -> str | None:
         xlst_image_location = os.getenv("XSLT_IMAGE_LOCATION", "")
         return self.body.content_html(f"{xlst_image_location}/{self.uri}")
+
+    def modified_xml(self) -> bytes:
```

Comment:
what do you think of 

```suggestion
    def xml_with_correct_frbr(self) -> bytes:
```

or something along those lines just so it is clear what is special about this xml property vs the normal one